doc/man1/ca.pod

```diff
@@ -701,6 +705,13 @@ For example if the CA certificate has:
 
 then even if a certificate is issued with CA:TRUE it will not be valid.
 
+=head1 NOTE
+
+Based on RFC 5280, since OpenSSL 1.1.1, the certificate validity period
```

Comment:
Call this HISTORY and start it as "Since OpenSSL version 1.1.1, the program follows RFC 5280 and ...."